src/routes/test/[slug]/api/submit-answer/+server.ts

```
@@ -64,7 +64,17 @@ export const POST: RequestHandler = async ({ request, cookies }) => {
 		);
 
 		if (!res.ok) {
-			throw new Error(`Failed to save answer: ${res.status} ${res.statusText}`);
+			let errorMessage = `Failed to save answer: ${res.status} ${res.statusText}`;
+
+			try {
+				const errorData = await res.json();
+				errorMessage = errorData?.detail || errorData?.message || errorData?.error || errorMessage;
+			} catch {}
```

ESLint

[error] Empty block statement.

(no-empty)